### ISSUE TITLE
Chromed Bird Windows Pops Open Randomly

### ISSUE THREAD

**Issue by jk0** [cezarsa/silver_bird#143]
From time to time the window will pop open at (what seems like) random times.

**Comment by rossburton** [comment]
I suspect that this is when CB refreshes.  For what it's worth, I only see this in OS X and not in Linux.

**Comment by cezarsa** [comment]
Yeah, I've seen this happen before, but it's quite rare, at least for me. So I've just spent more than an hour browsing through Chromium issues and trying to come up with a repro for this issue by creating an extension that would continuously receive calls from a background page. It all ended in nothing, nada... I'm on OS X using latest dev channel release (9.0.572.0 dev) and I wasn't able to reproduce it, not even once, either using CB or this test extension.
Do you guys have any idea? Are you able to reproduce it more easily?

**Comment by kurtraschke** [comment]
I've had this happen quite regularly with the Chrome dev channel on OS X for months now; it's actually quite a nuisance. I can't reproduce it on command, but Chromed Bird will randomly pop up, of its own accord.  If a Chrome incognito window or another application is in the foreground when this happens, then the Chromed Bird popup will actually appear on top of that window.

**Comment by okor** [comment]
Same here, only in OSx. No problems in Ubuntu 10.04 & 11.04.

I'm quite sure it's not random. Only happens when it updates and there is an unread tweet.

Reproduce it? Yeah, I just leave it on and it happens all the time.
